### ISSUE TITLE
Manual Download attempts result in white screen and no dl connection

### ISSUE THREAD

**Issue by bumpasticka** [Nexus-Mods/web-issues#983]
## Bug/error being reported:
My results when attempting a manual download in the last week or so have ranged from inconsistent success to white screen timeouts.  Doesn't seem to be a pattern as it happens with updated older and brand new files.  I thought I might have some unknown account issue.  Not the case as far as I can tell with no outstanding moderation issues.  Checked test download and ping.  All good.  See nothing like this happening to others in forums or I missed it.  Do not need any "standard answer" links to common download issues as this is something far different.  Can find no internal issues on my computer which, all things considered, is why the problem appears to be on your side.

## Steps to reproduce:
A white screen has been repeatedly generated on updates over the last few days for:
Micro Bikini
Dicky's Pinup Outfit
ASAL Vault Suit
Quantum ENB

White screen with more recent file 
Looksmenu Preset Lily (Aug 6 upload)

White screened just a minute ago:
My New Cait Preset (Aug 9 upload)

### Example:
See above

## Expected result:
Ability/working link for download versus apparent download restriction, misdirection or broken link.

**Comment by mrmason** [comment]
When you get a white screen please can you take the full URL of the file that is trying to be downloaded and reply to this issue with it ? The last part of the URL will have your IP in it, so feel free to remove that if you want to.

Can you also please provide your location, and the result of pinging `files.nexus-cdn.com` from your computer ?

**Comment by mrmason** [comment]
Also, are you a premium member, or supporter on the site - can you please provide me with your member id ?

**Comment by bumpasticka** [comment]
Sorry Tom!

I got all the info together yesterday and had everything done and listed here except for the ping results.  Went to pull those from the clipboard and paste here when....

Ended up having to update win 10 for clipboard updates during which I lost the ping results and just waited to calm down and "try again tomorrow."

Tomorrow comes and I have a new URL result across the board:  127.0.0.1:1001/status/204+IDM

I took this to be a bounce back and have something to do with Internet Download Manager.  I then tried a manual download that did not use the "default" IDM and success!

I have no way of recovering the original URLs so I guess my original issue remains a mystery.

I write this to thank you for your response and to point out that the handshake between IDM and Nexus appears to have issues.  I hope that info helps Nexus and other members out as a lot of us have to be using IDM.
